
## News

<div align="center">
  
<table border="0">
  <table>
    <tr>
    <td>
      <h2> A new integrated software named PhaBOX for phage identification and analysis has been developed! </h2>
      <p><b>Jan. 3rd, 2023</b></p>
      <p> Bacteriophages (phages) play key roles in regulating the composition/function of the microbiome by infecting their host bacteria. Lacking integrated software for phage identification and analysis, novel phages awaiting to be discovered constitute a large portion of “viral dark matter”. In this work, we developed a web server, named PhaBOX, to accurately identify and analyze phage contigs in metagenomic data. To our best knowledge, this is the first web server for comprehensive phage contig analysis in metagenomic data. PhaBOX integrates our previously published tools: PhaMer, PhaTYP, PhaGCN, and CHERRY, for phage identification, lifestyle prediction, taxonomy classification, and host prediction, respectively. To help users conduct downstream analysis, PhaBOX also provides visualization of the essential features for making the predictions, such as the similarity-based relationships between the contigs and other phages, predicted proteins on the contigs, and protein homology. All the predictions and intermediate results are provided for users. We hope that it can help advance the field of phage study in various ecosystems. To try PhaBOX, please click the link <a href="https://phage.ee.cityu.edu.hk/">PhaBOX</a> </p>
    </td>   
  </tr>  
  </table>
  <table>
    <tr>
    <td>
      <h2> Congratulations! Runzhou and Dehan’s work on viral genome assembly and polishing was accepted for publication by Bioinformatics.</h2>
      <p><b>Dec. 26th, 2022</b></p>
      <p>In this work, we introduce a new tool, AccuVIR, for viral genome assembly and polishing using error-prone long reads. It can better distinguish sequencing errors from true variants based on the key observation that sequencing errors can disrupt the gene structures of viruses, which usually have high density of coding regions. Our experimental results on both simulated and real third-generation sequencing data demonstrated its superior performance on generating more accurate viral genomes than generic assembly or polish tools. The tool is available at https://github.com/rainyrubyzhou/AccuVIR. Check out the paper at https://doi.org/10.1093/bioinformatics/btac827</p>
    </td>   
  </tr>  
  </table>
  <table>
    <tr>
    <td>
      <h2> Congratulations! Yilin, Jiayu and Peng Cheng's work on phage taxonomy classification was accepted for publication by Frontiers in Microbiology. </h2>
      <p><b>Dec. 22rd, 2022</b></p>
      <p> This is the first review conducted under the new ICTV classification framework since several large families were removed from ICTV in August 2022. This study provides a comprehensive review of phage classification in different scenarios and a practical guidance for choosing appropriate taxonomic classification pipelines. (https://www.frontiersin.org/articles/10.3389/fmicb.2022.1032186)</p>
    </td>   
  </tr>  
  </table>
       <table>
    <tr>
    <td>
      <h2> Congratulations! Dehan and Jiayu’s work on viral haplotype reconstruction was accepted for publication by Bioinformatics. </h2>
      <p><b>Oct. 21th, 2022</b></p>
      <p> This work developed a tool “HaploDMF” to reconstruct viral haplotypes from TGS data. Unlike existing tools that reconstruct haplotypes by checking the identity of overlap between reads, HaploDMF utilizes a deep matrix factorization model with an adapted loss function to automatically learn latent features from aligned reads. It is able to achieve highly robust performance on data with different properties while existing tools’ performance can be affected by the overlap size between reads. The tool is available at https://github.com/dhcai21/HaploDMF.</p>
    </td>   
  </tr>  
  </table>
     <table>
    <tr>
    <td>
      <h2> Congratulations! Jiayu's work on phage lifestyle prediction is accepted today in Briefings in Bioinformatics!</h2>
      <p><b>Oct. 17th, 2022</b></p>
      <p> Bacteriophages (or phages), which infect bacteria, have two distinct lifestyles: virulent and temperate. Predicting the lifestyle of phages helps decipher their interactions with their bacterial hosts, aiding phages' applications in fields such as phage therapy. PhaTYP adopt Bidirectional Encoder Representations from Transformer (BERT) to learn the protein composition and associations from phage genomes and achieves more stable performance on short contigs.
        arXiv version: [PhaTYP: Predicting the lifestyle for bacteriophages using BERT
](https://arxiv.org/abs/2206.09693)</p>
    </td>   
  </tr>  
  </table>
    <table>
    <tr>
    <td>
      <h2> Congratulations! Jiayu, Herui, Xubo, and Dehan received awards!</h2>
      <p><b>Sep 9, 2022</b></p>
      <p> They got the Research Tuition Scholarship ($42,096) and the Outstanding Academic Performance Award ($1,000) for their outstanding academic performance in the 2021-2022 academic year!</p>
    </td>   
  </tr>  
  </table>
  <table>
    <tr>
    <td>
      <h2> Congratulations! Jiayu's work on identifying phage is published today in Briefings in Bioinformatics!</h2>
      <p><b>Jun 30, 2022</b></p>
      <p><a href="https://doi.org/10.1093/bib/bbac258">Accurate identification of bacteriophages from metagenomic data using Transformer</a></p> 
    </td>   
  </tr>  
  </table>
      <table>
        <tr>
    <td>
      <h2> Congratulations! Herui and Dehan received the 2nd Prize in competition of "The 8th Hong Kong University Student Innovation and Entrepreneurship Competition" </h2>
      <p><b>June 07, 2022</b></p>
      <p> you can vist this website [https://www.hkchallengeplus.com/news/] for more information. </p>
    </td> 
    </tr>
    <tr>
    <td>
      <h2> Herui and Dehan attended the final round of the competition named "The 8th Hong Kong University Student Innovation and Entrepreneurship Competition" at Science Park of Hong Kong</h2>
      <p><b>May 27, 2022</b></p>
      <p> We recorded a video about our work in the competition. If you are interested in it, please scan this 2D code. </p>
    </td> 
    </tr>
    </table>
    <table>
    <tr>
    <td width="50%">
     <img src="/imgs/He_De_20220527.jpg" width = "400" height = "450" > 
    </td> 
     <td width="50%">
     <img src="/imgs/2D_code.jpg" width = "400" height = "400" >      
    </td> 
    </tr>
    </table>
    <table>
    <tr>
    <td>
      <h2> Congratulations! Jiayu's work on host prediction is published today in Briefings in Bioinformatics!</h2>
      <p><b>May 22, 2022</b></p>
      <p><a href="https://doi.org/10.1093/bib/bbac182">CHERRY: a Computational metHod for accuratE pRediction of virus–pRokarYotic interactions using a graph encoder–decoder model</a></p> 
    </td>   
  </tr>  
  </table>
    <table>
   <tr>
    <td>
      <h2> Congratulations! Xubo's poster presentation receives the $2,000 for 2nd Prize in EE faculty!</h2>
      <p><b>May 19, 2022</b></p>  
    </td>  
  </tr>
  </table> 
    <table>
    <tr>
    <td>
      <h2> Ph.D student Tang Xubo gave a talk titled "Sensitive RNA Virus Read Binning Using Learning-Based Models" </h2>
      <p><b>April 27, 2022</b></p>
      <p><a href="https://www.innocimda.com/Research_Oxford2022.html">The talk was held at a company called Centre for Intelligent Multidimensional Data Analysis Limited</a></p> 
    </td>   
  </tr>
  </table>
  <table width="100%">
    <tr>
    <td width="100%">
      <h2> Welcome！A new research assistant Liyan joined the group today </h2>
      <p><b>April 13, 2022  </b></p>
      <p>Welcome Liyan! </p>  
   </td>    
  </tr>
  </table>
  <table>
   <tr>
    <td>
      <h2>An invited talk: Charactering viral haplotypes using long reads </h2>
      <p><b>April 6, 2022  </b></p>
      <p>Yanni Sun will give an invited talk “Charactering viral haplotypes using long reads” at the Workshop on Combinatorial Problems of Strings and Graphs and Their Applications in Bioinformatics. This is organized by National University of Singapore and Institute for Mathematical Sciences.</p>
      <p><a href="https://ims.nus.edu.sg/events/combinatorial-problems-for-string-and-graph-and-their-applications-in-bioinformatics-part-2/">Combinatorial Problems of Strings and Graphs and Their Applications in Bioinformatics Part 2</a></p>   
   </td>    
  </tr>
   </table>
   <table>
   <tr>
    <td>
      <h2>Talk: Understanding the language of life using AI </h2>
      <p><b>Mar. 18, 2022  </b></p>
      <p>Yanni Sun gave a talk titled “Understanding the language of life using AI” to middle high-school students in Hong Kong at CityU-Learning Classroom. </p>  
    </td>
  </tr>
  </table>
  <table>
  <tr>
    <td>
      <h2>Dehan’s work is published</h2>
      <p><b>Feb. 14, 2022</b></p>
      <p>Dehan’s work on viral haplotype reconstruction “Reconstructing viral haplotypes using long reads” is published today!</p> 
        <p><a href="https://doi.org/10.1093/bioinformatics/btac089">Reconstructing viral haplotypes using long reads</a></p>  
    </td>   
  </tr>
  </table> 
  <table>
  <tr>
    <td>
      <h2>Xubo and Jiayu’s work on RNA virus classification is published</h2>
      <p><b>Feb. 7, 2022</b></p>
      <p>"RdRp-based sensitive taxonomic classification of RNA viruses for metagenomic data" is published today!</p>
      <p><a href="https://academic.oup.com/bib/article/23/2/bbac011/6523411?login=true">RdRp-based sensitive taxonomic classification of RNA viruses for metagenomic data</a></p>                                                                             
    </td>   
  </tr>
  </table> 
  <table>
 <tr>
    <td>
      <h2>Herui’s work on "strain-level" RNA virus composition analysis is published</h2>
      <p><b>Jan. 31, 2022</b></p>
      <p>"VirStrain: a strain identification tool for RNA viruses" is published!</p>
      <p><a href="https://doi.org/10.1186/s13059-022-02609-x">VirStrain: a strain identification tool for RNA viruses</a></p>               
    </td>   
  </tr>
  </table> 
</table>  
</div>
 <p><a href="/NUS_v3.pdf">The PDF format of a talk</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/yannisun/yannisun.github.io/raw/main/NUS_v3.pptx" download="NUS.pptx">The PPT format of a talk</a></p>
 


